src/collective/elastic/plone/restapi/expansions.py

```diff
@@ -74,5 +77,10 @@ def __call__(self, expand=False):
         # blocks_plaintext - only for Volto, not for ClassicUI
         # or Dexterity types w/o blocks behavior
         if IBlocks.providedBy(self.context):
-            result["blocks_plaintext"] = self._getBlocksPlainText(self.context)
+            result["collectiveelastic"].update(
```

Comment:
I already thought about it on my last refactoring session, but this breaks existing configurations. As a breaking change we need to bump the version to 3.0.0 in order to go forward.